src/building/menu.h

```diff
@@ -3,6 +3,40 @@
 
 #include "building/type.h"
 
+typedef enum {
+    BUILDING_MENU_VACANT_HOUSES = 0,
```

Comment:
Some building types are already prefixed with `BUILDING_MENU_*`, which is quite confusing. Suggestion: call this `BUILD_MENU_*` instead?

Reply:
Indeed, I had to add an `ALL` for those cases where the name was the same.

Your idea is simple and effective 👍